### PR TITLE
Additional Effects for standalone

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

**Review by Filemon279 — APPROVED** [review]

File: engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/FutureBaseStandaloneScenarioEngine.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+object FutureBaseStandaloneScenarioEngine {
+
+  type InterpreterType = StandaloneScenarioEngine.StandaloneScenarioInterpreter[Future]
+  private val scenarioTimeout = 10 seconds
+
+  implicit val cap: FixedCapabilityTransformer[Future] = new FixedCapabilityTransformer[Future]()
+
+  implicit val unwrapper: EffectUnwrapper[Future] = new EffectUnwrapper[Future] {
```
Comment (arkadius) [inline engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/FutureBaseStandaloneScenarioEngine.scala:18]: Is it "final" solution that this uwrapper is created statically? What about timeout configuration? If not, maybe some TODO?

File: docs/Changelog.md [hunk 2]
```diff
@@ -60,6 +60,8 @@ and `GenericNodeTransformation.implementation` is not invoked in this case
 * [#2465](https://github.com/TouK/nussknacker/pull/2465) aggregate-sliding emitWhenEventLeft parameter changed default value from true to false
 * [#2474](https://github.com/TouK/nussknacker/pull/2474) Interpreter return type changed from `F[Either[List[InterpretationResult], EspExceptionInfo[_ <: Throwable]]]` to `F[List[Either[InterpretationResult, EspExceptionInfo[_ <: Throwable]]]]`.
   Hence, e.g. multiple branches in Graph can be evaluated, both positively and negatively at the same time.
+* [#2540](https://github.com/TouK/nussknacker/pull/2540) It's possible to use different Effects than `Future` in standalone engine. `InvocationMetrics` are no longer
```
Comment (arkadius) [inline docs/Changelog.md:63]: can we stick with our normalized naming "request-response runtime" or "request-response interpreter" if is used as a librarary

**Review by arkadius — APPROVED** [review]

File: docs/Changelog.md (hunk 2 above)
Comment (mproch) [reply docs/Changelog.md]: Changed, though it's 1.1, so I left a mention of 'standalone', as rename is a part of 1.2 ;)

File: engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/FutureBaseStandaloneScenarioEngine.scala (hunk 1 above)
Comment (mproch) [reply engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/FutureBaseStandaloneScenarioEngine.scala:20]: Added TODO, though this timeout was fixed also in the previous solution. It's for test only

File: engine/standalone/engine/src/test/scala/pl/touk/nussknacker/engine/standalone/StandaloneProcessInterpreterSpec.scala [hunk 3]
```diff
@@ -308,7 +313,7 @@ class StandaloneProcessInterpreterSpec extends FunSuite with Matchers with Patie
 
     val interpreter = prepareInterpreter(process = process)
     val openApiOpt = interpreter.generateOpenApiDefinition()
-    println(openApiOpt.get)
+    //println(openApiOpt.get)
```
[the remark this answers is not in the document]
Comment (mproch) [reply engine/standalone/engine/src/test/scala/pl/touk/nussknacker/engine/standalone/StandaloneProcessInterpreterSpec.scala]: in the end, I just removed it...

**Pull request merged**